src/main/java/org/example/data/SuperPony.java

```diff
@@ -0,0 +1,24 @@
+package org.example.data;
+
+import java.util.List;
+import java.util.Random;
+
+public class SuperPony extends Pony implements Fly, Magic{
+
+    public SuperPony(String name, int xp, Coordinate position) {
+        super(name, xp, position);
+        List<String> magicCharacters = List.of("*", "#", "&");
+        Random random = new Random();
```

Comment:
this could be static final too, because you don't need a new random machine every time you make a super pony